conda/meta.yaml

```diff
@@ -8,6 +8,8 @@ source:
 
 build:
   script: python -m pip install --no-deps --ignore-installed .
+  entry_points:
+    - run_mtm2 = lsst.ts.m2.cli:run_mtm2
```

Comment:
The `run_mtm2` function is in the `csc` module and not the `cli` module so please correct this line.

Reply:
Fixed in the commit: [dc78758](https://github.com/lsst-ts/ts_m2/pull/25/commits/dc78758f743d2867d90bbb8500248fe6606a8870). Thanks!